tpl/collections/init.go

```diff
@@ -183,6 +183,16 @@ func init() {
 			},
 		)
 
+		ns.AddMethodMapping(ctx.Merge,
+			nil,
+			[][2]string{
+				{`{{ dict "title" "Hugo Rocks!" | collections.Merge (dict "title" "Default Title" "description" "Yes, Hugo Rocks!") }}`,
```

Comment:
@regisphilibert could I pick your brain about the argument order here ... I assume this is the most sensible, but the pipe syntax often throws me off the curve ... I have looked at the others and this makes sense to me, e.g. complement, "complement the slice on the left side of of the |".

Also, please shout if you hate names etc...

Reply:
Took me some time to get back into pipe + arg order... But If I am correct then the string above, without pipe would be like:

```
collections.Merge (dict "title" "Default Title" "description" "Yes, Hugo Rocks!") (dict "title" "Hugo Rocks!")
```

This is similar to every merge like functions of other languages I am aware of: **right** overrides **left**.

> Also, please shout if you hate names etc...

As long as its aliased `merge` like other `collections` methods, it's exactly what I would expect.

I'm very excited by this new method!

Reply:
Sure, thinking about this a little, I also suspect people will expect a case insensitive merge here, as in:

```
collections.Merge (dict "title" "Default Title" "description" "Yes, Hugo Rocks!") (dict "Title" "Hugo Rocks!")
```

That is what we do with configuration merges; it's a slight performance hit, but I suspect people (normally) will not use this for big data sets.